eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -1,12 +1,15 @@
+[CmdletBinding(SupportsShouldProcess)]
 param(
   # The repo owner: e.g. Azure
   $RepoOwner,
   # The repo name. E.g. azure-sdk-for-java
   $RepoName,
   # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
   $RepoId="$RepoOwner/$RepoName",
-  [Parameter(Mandatory = $true)]
-  $BranchPrefix,  
+  # CentralRepoId the original PR to generate sync PR. E.g Azure/azure-sdk-tools for eng/common
+  $CentralRepoId,
+  # We start from the sync PRs, use the branch name to get the PR number of central repo. E.g. sync-eng/common-(<branchName>)-(<PrNumber>). Have group name on PR number.
```

Comment:
Lets give an example regex with the PRNumber as the group name so it is clear how that is expected.